src/main/java/nl/tudelft/pixelperfect/event/EventFactory.java

```diff
@@ -35,38 +51,30 @@ public Event randomEvent() {
   /**
    * Create an event based on the parameter.
    *
-   * @param ind
+   * @param type
    *          Defines which event is to be created.
    * 
    * @return an Event.
    */
-  public Event create(int ind) {
-    Event tbr;
-    switch (ind) {
-      default:
-        tbr = null;
-        break;
-      case 0:
-        tbr = new AsteroidFieldEvent(id, "Asteroid Field",
-            "Watch out, you are approaching an asteroid field!", System.currentTimeMillis(), 4000,
-            10);
-        break;
-      case 1:
-        tbr = new FireEvent(id, "Fire", "Alert! Faulty wiring caused a fire!",
-            System.currentTimeMillis(), 4000, 10);
-        break;
-      case 2:
-        tbr = new HostileShipEvent(id, "Hostile Ship",
-            "A hostile spaceship is near, prepare to defend yourself!", System.currentTimeMillis(),
-            4000, 10);
-        break;
-      case 3:
-        tbr = new PlasmaLeakEvent(id, "Plasma Leak",
-            "Plasma pressure is dropping, there must be a leak!", System.currentTimeMillis(), 4000,
-            10);
-        break;
+  public Event create(int type) {
```

Comment:
That surprises me, I continuously regenerated the Maven site. Is the static error reported locally or in the Maven reporting process?